imls-backend/README.md

```diff
@@ -1,127 +1,116 @@
+
+```
+docker compose up
+```
+
+If this is the first time you are running our stack, a Postgrest (API) and Postgres (DB) base image will be downloaded before launch. The launch sequence involves the API container waiting for the DB container. When the DB container starts for the first time, it will load a substantial amount of SQL from the `init` directory. This *only* happens the first time the containers start up; on all subsequent restarts of the system, the `init` directory is not read. You can learn more about this [from the Postgres container documentation](https://hub.docker.com/_/postgres). (This behavior is built into the base image, and is not the result of our work.)
+
+### Optional: Loading test data
+
+At this point, you have a an empty system. It has no users and no data but is ready for use. If you're new to the stack, or want to do anything interesting at all, you're going to need some data and at least one user.
+
+`cd` to `imls-backend/test`.
+
+This directory has some test data to populate the database. **The data are not actually from the libraries named in the file**. Using invalid library IDs breaks many things, so there are *real* library IDs attached to *fake* data.
+
+We have a small script you can run to load that data.
+
+We recommend running this script as follows:
+
+```
+./setup-for-tests.sh
+```
+
+This will read in the needed environment variables for your containers, and then run a sequence of SQL commands to insert test data as well as create a user. That user is library `KY0069-002` and the API key for that user is `hello-goodbye`.
+
+## Testing everything
+
+You should now be able to run the unit tests. This is a good test of whether your stack is functional. The tests in this file will only pass if you have loaded the test data in the previous step.
+
+Poetry will read the `pyproject.toml` file and install all dependencies into a virtual environment with the command (issued from the imls-backend/ folder)
+
+```
+poetry install
+```
+
+After this, you can drop into the virtual environment with the command
+
+```
+poetry shell
+```
+
+Now run the tests. Note there is a `.test-config` file in the `test/` directory. This file supplies environmental variables specific to running the `pytest` tests, and needs to be sourced in the shell where `pytest` is run.
+
+```
+source test/.test-config && pytest test/
+```
+
+If all goes well, you'll see all the tests pass.
+
+### Optional: Using a DB browser
+
+There are many options for DB browsers. DBeaver is a good optoin, and the community edition will work just fine.
+
+[Instructions for installing DBeaver](https://dbeaver.io/download/).
+
+You will need to create a new database connection. The connection parameters will be the same as in the `.env` file in the imls-backend/ folder. If you did not modify the environment variables, you should be able to use the username `postgres`, database `imls`, and password `imlsimlsimls`.
+
+## What now?
```

Comment:
Think we can remove this section for the purposes of straightforward dev docs

Reply:
Same thing here, moved it to DATABASE_NOTES.md